modules/@apostrophecms/image/ui/apos/components/AposMediaUploader.vue

```diff
@@ -1,17 +1,16 @@
 <template>
   <label
     :class="dropzoneClasses"
-    :disabled="disabled"
+    :disabled="disabled ? disabled : null"
```

Comment:
ensures `disabled` is not attributed to element when disabled is `false`

Reply:
```suggestion
    :disabled="disabled || null"
```

optional